style.css

```diff
@@ -135,3 +150,158 @@ ul li {
 }
 
 /* Home-page Ends here */
+
+/* work-section starts here */
+
+.grid-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  place-items: center;
+}
+
+#work-section {
+  margin-top: 70px;
+  height: auto;
+  padding-top: 160px;
+  padding-bottom: 160px;
+  background: var(--portfolio-color);
+  position: relative;
+  z-index: -10;
+}
+
+.card {
+  width: 327px;
+  height: 568px;
+  background: var(--card-color);
+  border: 1px solid var(--card-border);
+  border-radius: 16px;
+  padding: 15px;
+  margin-bottom: 4rem;
+}
+
+.card-content {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.card-content h1 {
+  font-size: 28px;
+  line-height: 38px;
+  margin-top: 0.5rem;
+}
+
+.card-content h3 {
+  color: var(--canopy-color);
+}
+
+.card-content a {
+  color: var(--work-details);
+}
+
+.work-details {
+  display: flex;
+  align-items: center;
+  gap: 0.5em;
+  margin-top: 0.5rem;
+}
+
+.bullet-point {
+  background: var(--work-details);
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+}
+
+.card-content p {
+  margin-top: 1rem;
+}
+
+.work-tools {
+  margin-top: 0.2rem;
+}
+
+.tools {
+  color: var(--logo-color);
+  background: var(--work-tools);
+  border: none;
+  outline: none;
+  padding: 8px 12px;
+  border-radius: 8px;
+  font-size: 14px;
+}
+
+button {
+  margin-top: 1rem;
+  padding: 9px 20px;
+  color: var(--logo-color);
+  border: 1px solid var(--logo-color);
+  background: var(--card-color);
+  font-size: 16px;
+  border-radius: 8px;
+  cursor: pointer;
+}
+
+button:hover {
+  background: var(--hover-color);
+  color: var(--card-color);
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+}
+
+/* About-section starts here */
+#about-section {
+  width: 375px;
+  height: 1263;
+  margin-top: -70px;
+  background-image: url("./images/about-base.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.about-title h1 {
+  margin-top: 100px;
+}
+
+.about-title p {
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+}
+
+.languages-section h3 {
+  color: var(--l-color);
+  margin-top: 2.5rem;
+}
+
+.languages-title li {
+  display: flex;
+  justify-content: space-between;
+}
+
+.languages-title img {
+  margin-top: 2.7rem;
+  margin-right: 2rem;
+}
+
+.languages-box {
+  background: var(--box-color);
+  margin-top: 2rem;
```

Comment:
:red_circle: The space between the languages-box items should be the same as on the Figma design. You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).Please reduce this margin to reflect something closer to the Figma design. Something like` 1rem `can make the deal :hugs: 
**Your Design**
![281](https://user-images.githubusercontent.com/26736582/198398843-16b6885b-9154-406c-a5e1-78667517ff0f.png)
**Figma Design**
![282](https://user-images.githubusercontent.com/26736582/198398897-d113728e-cdb4-4ec6-96ae-c30bc242d572.png)